pages/common/az-account.md

```diff
@@ -0,0 +1,24 @@
+# az account
+
+> Manage Azure subscription information.
+> More information: <https://docs.microsoft.com/cli/azure/account>.
+
+- Print a list of subscriptions for the logged in account:
+
+`az account list`
+
+- Set a `subscription` to be the currently active subscription:
```

Comment:
```suggestion
- Set a `subscription` to be the active subscription:
```

Reply:
This would need to be applied throughout if agreed on

Reply:
I think both are fine, but maybe with `currently` it reads a bit more fluently.

Reply:
> This would need to be applied throughout if agreed on

Yep, let me send in a PR for the one that we have merged already :)

Reply:
IMO `currently` reads better .. for now going with that :)